1-exercises/A-array-find/exercise.js

```diff
@@ -14,10 +14,15 @@ let names = [
   "Mikey",
   "Anastasia",
   "Karim",
-  "Ahmed",
+  "Ahmed"
 ];
 
-let longNameThatStartsWithA = findLongNameThatStartsWithA(names);
+function isLongNameThatStartsWithA(name) {
+return name.length > 7 && name.charAt(0) === "A";
+// I tried both (return name.length > 7) alone and with the expression after && and they returned the same result
```

Comment:
Good observation. If you were writing tests you would want an example that tested both conditions.